packages/mattermost-redux/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "mattermost-redux",
```

Comment:
I don't think these are needed if the components package.json doesn't consider this as a workspace

Reply:
But it does so we need it.